### ISSUE TITLE
Next/prev do not work after editing people (removing all original people)

### ISSUE THREAD

**Issue by fbristow** [immich-app/immich#13241]
### The bug

TL;DR: After editing the people in a photo, navigating to next and previous does not work when all *original* people have been removed.

I am in a state where I have two children who have been partially assigned to the same "person" (i.e., I have a child "A" and a child "B"; some photos have been assigned to child "A", some to child "B", but many to child "A+B" with multiple instances of "A+B" in the same photo).

I want to be able to navigate to the next photo because I then otherwise need to go back to the "Explore" page for "A+B", refresh, then select a new photo. This is the behaviour I saw before and would like to continue to experience.

### The OS that Immich Server is running on

Latest on Docker.

### Version of Immich Server

v1.117.0

### Version of Immich Mobile App

v1.117.0

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
name: immich

services:
  immich-server:
    container_name: immich_server
    image: ghcr.io/immich-app/immich-server:${IMMICH_VERSION:-release}
    # extends:
    #   file: hwaccel.transcoding.yml
    #   service: cpu # set to one of [nvenc, quicksync, rkmpp, vaapi, vaapi-wsl] for accelerated transcoding
    volumes:
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
      - /etc/localtime:/etc/localtime:ro
    env_file:
      - .env
    ports:
      - 2283:3001
    depends_on:
      - redis
      - database
    restart: always

  immich-machine-learning:
    container_name: immich_machine_learning
    # For hardware acceleration, add one of -[armnn, cuda, openvino] to the image tag.
    # Example tag: ${IMMICH_VERSION:-release}-cuda
    image: ghcr.io/immich-app/immich-machine-learning:${IMMICH_VERSION:-release}
    # extends: # uncomment this section for hardware acceleration - see https://immich.app/docs/features/ml-hardware-acceleration
    #   file: hwaccel.ml.yml
    #   service: cpu # set to one of [armnn, cuda, openvino, openvino-wsl] for accelerated inference - use the `-wsl` version for WSL2 where applicable
    volumes:
      - model-cache:/cache
    env_file:
      - .env
    restart: always

  redis:
    container_name: immich_redis
    image: docker.io/redis:6.2-alpine@sha256:328fe6a5822256d065debb36617a8169dbfbd77b797c525288e465f56c1d392b
    healthcheck:
      test: redis-cli ping || exit 1
    restart: always

  database:
    container_name: immich_postgres
    image: docker.io/tensorchord/pgvecto-rs:pg14-v0.2.0@sha256:90724186f0a3517cf6914295b5ab410db9ce23190a2d9d0b9dd6463e3fa298f0
    environment:
      POSTGRES_PASSWORD: ${DB_PASSWORD}
      POSTGRES_USER: ${DB_USERNAME}
      POSTGRES_DB: ${DB_DATABASE_NAME}
      POSTGRES_INITDB_ARGS: '--data-checksums'
    volumes:
      - ${DB_DATA_LOCATION}:/var/lib/postgresql/data
    healthcheck:
      test: pg_isready --dbname='${DB_DATABASE_NAME}' --username='${DB_USERNAME}' || exit 1; Chksum="$$(psql --dbname='${DB_DATABASE_NAME}' --username='${DB_USERNAME}' --tuples-only --no-align --command='SELECT COALESCE(SUM(checksum_failures), 0) FROM pg_stat_database')"; echo "checksum failure count is $$Chksum"; [ "$$Chksum" = '0' ] || exit 1
      interval: 5m
      start_interval: 30s
      start_period: 5m
    command: ["postgres", "-c" ,"shared_preload_libraries=vectors.so", "-c", 'search_path="$$user", public, vectors', "-c", "logging_collector=on", "-c", "max_wal_size=2GB", "-c", "shared_buffers=512MB", "-c", "wal_compression=on"]
    restart: always

volumes:
  model-cache:
```

### Your .env content

```Shell
# You can find documentation for all the supported env variables at https://immich.app/docs/install/environment-variables

# The location where your uploaded files are stored
UPLOAD_LOCATION=./library
# The location where your database files are stored
DB_DATA_LOCATION=./postgres

# To set a timezone, uncomment the next line and change Etc/UTC to a TZ identifier from this list: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List
# TZ=Etc/UTC

# The Immich version to use. You can pin this to a specific version like "v1.71.0"
IMMICH_VERSION=release

# Connection secret for postgres. You should change it to a random password
DB_PASSWORD=postgres

# The values below this line do not need to be changed
###################################################################################
DB_USERNAME=postgres
DB_DATABASE_NAME=immich
```

### Reproduction steps

1. Starting from an environment where a database contains several photos with people assigned to faces.
2. Navigate to "Explore" &rarr; choose a person that has many photos.
3. Select a photo from the collection.
4. Click on :pencil2: next to "People".
5. Change all people in the photo and click  "Done" (**leaving no original people**).
6. At this point I cannot navigate to next or prev (with keyboard or the buttons on screen) because this photo no longer belongs to the collection.

**Comment by abhishek-singh0710** [comment]
Hi, I'm quite new to open-source and would like to work on this issue. Could you kindly tell me in more detail how to reproduce this, I couldn't understand the first step in the reproduction steps part.
Thanks

**Comment by fbristow** [comment]
Hi @abhishek-singh0710, this is more straightforward to reproduce than I thought: if you have people attached to the photo, edit the photo, then remove all originally assigned people, then you can't navigate through the collection with next and prev anymore. I've updated the description.

**Comment by danieldietzler** [comment]
@abhishek-singh0710 are you still interested in working on this?

**Comment by Pranay-Pandey** [comment]
I think this is fixed now. I am not able to reproduce this.

**Comment by fbristow** [comment]
I've upgraded to 1.118.2 and I am still able to reproduce this issue.

**Comment by danieldietzler** [comment]
@fbristow any updates? 👀 
There have also been more changes in that area so it's possible it's fixed by now?

**Comment by fbristow** [comment]
This appears to be fixed in [v1.131.3](https://github.com/immich-app/immich/releases/tag/v1.131.3). Closing. Thank you!
